### ISSUE TITLE
POC of OpenRewrite JPMML rule

### ISSUE THREAD

**Issue by gitgabrio** [IBM/kie-roadmap#16]

**Comment by baldimir** [comment]
I thought about this a bit. Another option would be to extract the PMML modules to a separate repository. As far as I heard, not many users are using it. What do you think please? I mean instead of investing into maintenance component, moving it elsewhere and make it up to date if someone requests it.

**Comment by gitgabrio** [comment]
The problem is not only for kie-pmml-trusty - also kie-dmn module uses directly jpmml-model. So, the problem will stay the same

**Comment by baldimir** [comment]
Yes, but is anyone using this integration? Calling PMML from DMN.

**Comment by gitgabrio** [comment]
Yes, this happen - I've been asked different times, in the last years, about that - just two days ago for the last time

**Comment by baldimir** [comment]
The PR is done to the Tristan's repository - https://github.com/radtriste/drools/pull/28

**Comment by gitgabrio** [comment]
Fixed by https://github.com/radtriste/drools/pull/32
